Makefile

```diff
@@ -96,22 +94,25 @@ check-go-format: prebuild-check
 	&& exit 1 \
 	|| true
 
-.PHONY: golint
-.ONESHELL: golint
+.PHONY: analyze-go-code
+.ONESHELL: analyze-go-code
```

Comment:
No need for ONESHELL here since you're not executing a recipe but just dependencies.